set.mm

```diff
@@ -150869,6 +150898,25 @@ Finite intervals of nonnegative integers (or "finite sets of sequential
     cle ltle sylbi ) ACBDEFAGFZBHFZABIJZKABSJZABLUBUCUDUEUBAMFBMFUDUENUCAOBPABT
     QRUA $.
 
+  $( The difference of the upper bound of a half-open range of nonnegative
+     integers and an element of this range is greater than or equal to 1.
+     (Contributed by AV, 1-Sep-2025.) $)
+  elfzo0subge1 $p |- ( A e. ( 0 ..^ B ) -> 1 <_ ( B - A ) ) $=
+    ( cc0 cfzo co wcel cn0 cn clt wbr w3a c1 cmin cle elfzo0 caddc cz wb nn0z
+    cr nnz zltp1le syl2an biimp3a nn0re 3ad2ant1 1red 3ad2ant2 leaddsub2d mpbid
+    nnre sylbi ) ACBDEFAGFZBHFZABIJZKZLBAMENJZABOUPALPEBNJZUQUMUNUOURUMAQFBQFUO
+    URRUNASBUAABUBUCUDUPALBUMUNATFUOAUEUFUPUGUNUMBTFUOBUKUHUIUJUL $.
```

Comment:
(pushable to later)

shorter proof

```
    ( cc0 cfzo co wcel c1 elfzoelz zred elfzoel2 1red clt wbr cmin cle elfzolt2
    zltlem1d mpbid lesubd ) ACBDEFZABGTAACBHZITBACBJZITKTABLMABGNEOMACBPTABUAUB
    QRS $.
```

(shorter proof of elfzolem1) is a subproof of the above